### PR TITLE
Scroll area

### PR DESCRIPTION
**Pull request by kof**
## Description

1. Add scroll area to the to ds
2. Add scroll area to add components panel

## Code Review

- [ ] hi @TrySound , I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: apps/builder/app/builder/features/sidebar-left/panels/components/scroll-area.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  padding: 2,
+  background: theme.colors.blackA6,
+  transition: "background 160ms ease-out",
+  "&:hover": { background: theme.colors.blackA8 },
+  '&[data-orientation="vertical"]': { width: SCROLLBAR_SIZE },
+  '&[data-orientation="horizontal"]': {
+    flexDirection: "column",
+    height: SCROLLBAR_SIZE,
+  },
+});
```
Comment (TrySound) [inline apps/builder/app/builder/features/sidebar-left/panels/components/scroll-area.tsx:40]: Doesn't radix provide all necessary style?

Comment (kof) [reply apps/builder/app/builder/features/sidebar-left/panels/components/scroll-area.tsx:40]: no, I will move it to design-system, radix never provides styles

Comment (kof) [reply apps/builder/app/builder/features/sidebar-left/panels/components/scroll-area.tsx:40]: these styles are copied from example, I will update them to use native macos styles

Comment (kof) [reply apps/builder/app/builder/features/sidebar-left/panels/components/scroll-area.tsx:40]: or Taylor will design something

File: packages/design-system/src/components/scroll-area.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    height: SCROLLBAR_SIZE,
+  },
+});
+
+type ScrollAreaProps = { children: JSX.Element | Array<JSX.Element> };
+
+export const ScrollArea = ({ children }: ScrollAreaProps) => {
+  return (
+    <ScrollAreaRoot>
+      <ScrollAreaViewport> {children}</ScrollAreaViewport>
```
Comment (TrySound) [inline packages/design-system/src/components/scroll-area.tsx:68]: Unnecessary space?

**Review by TrySound — APPROVED** [review]

File: pnpm-lock.yaml [hunk 3]
```diff
@@ -5618,6 +5621,18 @@ packages:
       react-dom: 17.0.2(react@17.0.2)
     dev: false
 
+  /@radix-ui/react-primitive@1.0.2(react-dom@17.0.2)(react@17.0.2):
```
Comment (TrySound) [inline pnpm-lock.yaml:5624]: The third instance of react-primitive. We will need to bump radix packages.

Comment (kof) [reply pnpm-lock.yaml:5624]: lets bump

Comment (kof) [reply pnpm-lock.yaml:5624]: separate pr for bumping or same?

**Pull request merged**